src/InteractiveAtom.tsx

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
 
+import { css } from 'emotion';
+import { space } from '@guardian/src-foundations';
+import { neutral, text } from '@guardian/src-foundations/palette';
+
 import { InteractiveAtomType } from './types';
 
-export const InteractiveAtom = ({ id }: InteractiveAtomType): JSX.Element => (
-    <div data-atom-id={id}>InteractiveAtom</div>
+export const InteractiveAtom = ({
+    id,
+    url,
+}: InteractiveAtomType): JSX.Element => (
+    <div
+        data-atom-id={id}
+        data-atom-type="interactive"
+        className={css`
+            padding-bottom: ${space[1]}px;
+            padding-left: ${space[2]}px;
+            padding-right: ${space[2]}px;
+            border-top: 1px solid ${text.primary};
```

Comment:
this isnt present on Frontend, the atom seems to automatically come with a grey to border, so no need to add one